src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAO.scala

```diff
@@ -49,12 +50,34 @@ object VertexSchema {
   }
 }
 
+object EdgeSchema {
+  val sep = "_"
+  sealed trait EdgeRelationType {
+    def prefix: String
+    def toLabel(label:String) = prefix + sep + label
+  }
+  case object Own extends EdgeRelationType {
+    override val prefix = "OWN"
+  }
+  case object Ref extends EdgeRelationType {
+    override val prefix = "REF"
+  }
+  val allEdgeRelations = Seq(Own, Ref)
+
+  def getEdgeRelation(label: String): EdgeRelationType = {
+    toEdgeRelation(label.split(sep).head)
+  }
+
+  def toEdgeRelation(prefix: String): EdgeRelationType = {
+    allEdgeRelations.filter( er => prefix == er.prefix ).head
+  }
+  def stripEdgeRelation(str: String): String = str.split(sep, 2).last
```

Comment:
sigh. no. It _removes_ the relation type from the edge label. An edge label `foo` gets rewritten to `OWN_foo`. This function returns the "original name" of the label, i.e. without the `OWN_`.